initrd/bin/oem-factory-reset

```diff
@@ -450,7 +496,7 @@ if [ $GPG_EXPORT -ne 0 ]; then
         ERROR=$(tail -n 1 /tmp/error | fold -s)
         whiptail_error_die "Key export error: unable to copy ${GPG_GEN_KEY}.asc to /media:\n\n$ERROR"
     fi
-    umount /media 2>/dev/null
+    mount -o remount,ro /media 2>/dev/null
```

Comment:
why remount here? we're going to reboot immediately after

Reply:
@MrChromebox the initial script took for granted it was ran from ramfs, which assumption didn't fit my use case.

The reason it was unmounted is because we lack flush support, where everywhere else in code, we make sure changes are flushed by remounting read only.

This is why /media is also mounted only if not mounted already, and if mounted already, remounted in rw to modify content and then here in ro.